src/main/java/pw2021/backend/Flatly/enities/User.java

```diff
@@ -0,0 +1,120 @@
+package pw2021.backend.Flatly.enities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    private String login;
+    private String password;
+    private String firstName;
+    private String secondName;
+    private String securityToken;
```

Comment:
Mamy tę klasę SecurityToken, którą joinujemy z Userem. Po co User ma pole securityToken typu string? Czy ono nie powinno być typu SecurityToken?

Reply:
Zapomniałem usunąć pola typu string, jednakże to już zmergowany pull request i trzeba to zrobić w przyszłości przy okazji

Reply:
Dzięki za odpowiedź